teams/infra-bors.toml

```diff
@@ -0,0 +1,10 @@
+name = "infra-bors"
```

Comment:
Nit: is "bors-infra" a more appropriate name? As it is, it reads like we're trying to indicate this is a subteam of infra (which is not the highest priority information) while "bors-infra" seems to indicate what the team is (i.e., the team in charge of the infrastructure related to bors).

Reply:
There was already `infra-admins`, so I took inspiration in that. I think that the naming `higher level -> lower level` makes sense, but I'm not opposed to changing it.

Reply:
Most similar teams are named with parent prefix (e.g., libs-contributors, compiler-contributors, etc.)

Renaming should be cheap down the line in any case.